shared_resources/json/en.lproj/walk_section_step.json

```diff
@@ -0,0 +1,135 @@
+        "type": "instruction",
+        "title": "Walk for 30 seconds",
+        "subtitle": "You will be walking continuously at your normal pace for 30 seconds. Try to walk in a straight path. If you need to turn around then turn around as if you are walking around a cone.",
+        "fullInstructionsOnly": true,
+        "image": {
+            "type": "animated",
+            "imageNames": ["walking_1", "walking_2", "walking_3", "walking_4", "walking_5", "walking_6", "walking_7", "walking_8", "walking_9", "walking_10", "walking_11", "walking_12"],
+            "placementType": "topMarginBackground",
+            "animationDuration": 6.0
+        },
+        "actions": {
+            "goForward": {
+                "type": "default",
+                "buttonTitle": "Got it"
+            }
+        }
+    },
+    {
+        "identifier": "volumeUp",
+        "type": "instruction",
+        "title": "Turn the volume up",
+        "subtitle": "Bring your phone with you and turn up your phone volume so you can hear the instructions while you are walking.",
+        "image": {
+            "type": "fetchable",
+            "imageName": "turn_up_volume",
+            "placementType": "topMarginBackground"
+        },
+        "actions": {
+            "goForward": {
+                "type": "default",
+                "buttonTitle": "The phone’s volume is up"
+            }
+        }
+    },
+    {
+        "identifier": "pantsWithPocket",
+        "type": "instruction",
+        "title": "Pants with front pockets",
+        "subtitle": "Before continuing on with the short walk, please make sure you have front pockets so you have a place to put the phone. If you do not have pockets you can place the phone in the waistband of your pants.",
+        "image": {
+            "type": "fetchable",
+            "imageName": "pants_w_pocket",
+            "placementType": "topBackground"
+        },
+        "actions": {
+            "goForward": {
+                "type": "default",
+                "buttonTitle": "Got front pockets"
+            }
+        }
+    },
+    {
+        "identifier": "phoneInPocket",
+        "type": "instruction",
+        "title": "Put the phone in your pocket",
+        "image": {
+            "type": "animated",
+            "imageNames": ["phone_in_pocket_1", "phone_in_pocket_2", "phone_in_pocket_3", "phone_in_pocket_4"],
+            "placementType": "topMarginBackground",
+            "animationDuration": 2,
+            "animationRepeatCount": 1
+        },
+        "colorMapping": {
+            "type": "singleColor",
+            "colorStyle": "white"
+        },
+        "subtitle": "This test measures your leg movement and you will need to put your phone in your front pocket. If you do not have pockets you can place the phone in the waistband of your pants.",
+        "actions": {
+            "goForward": {
+                "type": "default",
+                "buttonTitle": "Putting it in my pocket"
```

Comment:
I thought we'd moved to not using custom titles for the buttons and just using "Next"?